### PR TITLE
Complete captone project

### PR DESCRIPTION
**Pull request by afizsavage**
1. Add timeline section
2. Add main event section
3. Add about page
4. Make index and about page responsive

Here is the project presentation [video](https://www.loom.com/share/9ae0100c5d41470c934e47a6769e5b97).

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+          <ul>
+            <li><a href="./index.html">Home</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li>
+          </ul>
+          <button type="button">EC Campaign</button>
+        </div>
+      </nav>
```
Comment (dasileker) [inline about.html]: - [ ] As I said in the video the navbar should contain two columns.

- **Your design**

Opera Instantané_2021-11-19_123805_localhost


- **Behance design**

Capture web_19-11-2021_123833_www behance net

File: index.html [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+          <ul>
+            <li><a href="./about.html">About</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li>
+          </ul>
+          <button type="button">SWM Campaign</button>
+        </div>
+      </nav>
```
Comment (dasileker) [inline index.html]: - [ ] As I said in the video the navbar should contain two columns.

- **Your design**

Opera Instantané_2021-11-19_123805_localhost


- **Behance design**

Capture web_19-11-2021_123833_www behance net

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @afizsavage,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

- Please look at the video and implement the changes.


https://user-images.githubusercontent.com/61562626/142617650-db7f31f1-2d04-4116-874b-20520413aff6.mp4



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by VanessaAoki — APPROVED** [review]
## STATUS: APPROVED 🟢 
Hi @afizsavage, 

Great job implementing all the changes required by the previous Code Reviewer, your project is complete! There is nothing else to say other than... it's time to merge it! 
Congratulations! 🎉 

approved

### Project highlights:
- [x] All linter checks are passing :heavy_check_mark: 
- [x] Your README file looks very professional and the PR has a descriptive title/summary of the implemented changes :memo: 
- [x] Descriptive commits and correct use of the GitHub Flow :fire: 


Cheers and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**